File: app/src/main/java/comexpensetracker/medium/extra_expensetracker/MessageResponse.java
package comexpensetracker.medium.extra_expensetracker;

import com.google.gson.annotations.SerializedName;

/**
 * Created by akash on 4/7/17.
 */

public class MessageResponse {

    @SerializedName("message")
    String message;

    public String getMessage() {
        return message;
    }

}

File: app/src/main/java/comexpensetracker/medium/extra_expensetracker/MessageResponse.kt
package comexpensetracker.medium.extra_expensetracker

import com.google.gson.annotations.SerializedName



/**
 * Created by akash on 23/7/17.
 */
class MessageResponse {

    @SerializedName("message")
    var message: String? = null

    fun getMessage(): String? {
        return message
    }

}
